src/Datagrid/ProxyQueryInterface.php

```diff
@@ -94,11 +94,19 @@ public function setMaxResults($maxResults);
     public function getMaxResults();
 
     /**
+     * NEXT_MAJOR: Remove this method.
```

Comment:
this method from the interface.

?

Otherwise it could be a bit misleading IMO

Reply:
I've changed it. I think if we usually put `NEXT_MAJOR` comments next to the every file affected, so I usually understand that if there is no more instructions than `Remove this method`, it's to remove the method where the `NEXT_MAJOR` comment is.

BTW I had forgotten to add the comment to the implementation in tests 😬